ods_ci/tests/Tests/100__deploy/100__installation/112__model_registry.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
+Documentation       Post install test cases that verify Model Registry
+Library             OpenShiftLibrary
+Resource            ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Resource            ../../../Resources/OCP.resource
+
+
+*** Variables ***
+${MODEL_REGISTRY_NS}=    ${APPLICATIONS_NAMESPACE}
+#Default DSC for ODH only, need to be changed for RHOAI later
+${DSC_NAME}=             default
+
+*** Test Cases ***
+Verify Model Registry Operator Installation
+    [Documentation]    Verifies that the Model Registry operator has been
+    ...    deployed in the ${APPLICATIONS_NAMESPACE} namespace in ODS
+    [Tags]    OpenDataHub    robot:recursive-continue-on-failure
+    ${modelregistry} =    Is Component Enabled    modelregistry    ${DSC_NAME}
+    IF    "${modelregistry}" == "false"
+    Enable Component    modelregistry
+    END
```

Comment:
if it was disabled, then you should disable it in test teardown

Reply:
Added Teardown test now.